src/plone/restapi/tests/test_documentation.py

```diff
@@ -1536,16 +1536,54 @@ def test_documentation_expansion(self):
         response = self.api_session.get("/front-page?expand=breadcrumbs,workflow")
         save_request_and_response_for_docs("expansion", response)
 
-    @unittest.skipIf(not PLONE5, "Just Plone 5 currently.")
+
+@unittest.skipIf(not PLONE5, "Just Plone 5 currently.")
+class TestControlPanelDocumentation(TestDocumentationBase):
+
+    layer = PLONE_RESTAPI_DX_FUNCTIONAL_TESTING
+
     def test_controlpanels_get_listing(self):
         response = self.api_session.get("/@controlpanels")
         save_request_and_response_for_docs("controlpanels_get", response)
 
-    @unittest.skipIf(not PLONE5, "Just Plone 5 currently.")
     def test_controlpanels_get_item(self):
         response = self.api_session.get("/@controlpanels/editing")
         save_request_and_response_for_docs("controlpanels_get_item", response)
 
+    def test_controlpanels_get_dexterity(self):
+        response = self.api_session.get("/@controlpanels/dexterity-types")
+        save_request_and_response_for_docs("controlpanels_get_dexterity", response)
+
+    def test_controlpanels_crud_dexterity(self):
+        # POST
+        response = self.api_session.post(
+            "/@controlpanels/dexterity-types",
+            json={
+                "title": "My Custom Content Type",
+                "description": "A custom content-type",
+            },
+        )
+        save_request_and_response_for_docs("controlpanels_post_dexterity_item", response)
+
+        # GET
```

Comment:
Those should go into individual tests IMHO to indicate what's wrong to the developers in case we have a test failure.

Reply:
Oh. Those are just documentation generation tests. Didn't realize that at first. I'd still prefer individual test functions here I guess.

Reply:
Inspired from https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/tests/test_documentation.py#L234

But I suppose I can split it :wink: